meteor/client/styles/shelf/dashboard.scss

```diff
@@ -390,31 +389,58 @@ $dashboard-button-height: 5.625em;
 			top: 0;
 			right: 0;
 			bottom: 0;
+			background: linear-gradient(90deg, rgb(84,84,84) 0%, rgb(61,61,61) 100%);
 
 			> .dashboard-panel__panel__button__label-container {
-				position: relative;
-				left: 0;
-				top: 0;
-				right: 0;
-				bottom: 0;
-				padding: 3px 4px;
-				z-index: 2;
+				background-color: #000;
+				height: $dashboard-button-label-height;
+				display: flex;
+				position: absolute;
+				top: $dashboard-button-thumbnail-height;
+
+				.dashboard-panel__panel__button__tag-container {
+					flex: 0 0 0.46875em;
+					z-index: 2;
+					background: var(--segment-layer-background-unknown);
+
+					&.camera {
+						background: var(--segment-layer-background-camera);
+					}
+
+					&.graphics {
+						background: var(--segment-layer-background-graphics);
+					}
+
+					&.live-speak {
+						background: var(--segment-layer-background-live-speak);
+					}
+
+					&.lower-third {
+						background: var(--segment-layer-background-lower-third);
+					}
+
+					&.remote {
+						background: var(--segment-layer-background-remote);
+					}
+
+					&.vt {
+						background: var(--segment-layer-background-vt);
```

Comment:
This can be done easier/more reliably using `item-type-colors()` from https://github.com/nrkno/sofie-core/blob/031e3e3da71a9dc0eb6c4b15215eb375fc5e08ec/meteor/client/styles/_itemTypeColors.scss#L69-L79